rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,40 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+linter for the major programming languages the repo is written in.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked this
+can lead to inconsistencies in style, which over time can make code messy, and
+difficult to maintain and review. It can also lead to bikeshedding over code
+style in code reviews, which can cause conflict and slow down code review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories SHOULD use an
+opinionated code linter and formatter to make sure code meets a standard style.
+Examples of this include (but are not limited to):
+
+- [Standard.rb](https://github.com/testdouble/standard)
+- [Standard for Javascript](https://github.com/standard/standard)
+- [Standard for Typescript](https://github.com/standard/ts-standard)
+- [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer)
+- [Black](https://github.com/psf/black)
```

Comment:
Super minor thing: I think it's worth appending `(for Python)` to the end of the Black line -- every other one is semi-obvious which language it's for.

Reply:
Good call 👍